src/backend/aspen/app/views/sample.py

```diff
@@ -416,3 +416,60 @@ def create_sample():
     except Exception as e:
         raise ex.BadRequestException(f"Error encountered when saving data: {e}")
     return jsonify(success=True)
+
+
+@application.route("/api/samples/update/publicids", methods=["POST"])
+@requires_auth
+def update_sample_public_ids():
+    user: User = g.auth_user
+    request_data = request.get_json()
```

Comment:
Should we check that the user is a system admin here?

Reply:
added